File: ruby/view/tableprogressindicator.rb
# Created by Satoshi Nakagawa.
# You can redistribute it and/or modify it under the Ruby's license or the GPL2.

class TableProgressIndicator < OSX::NSProgressIndicator
  include OSX
  
  def mouseDown(e)
    self.superview.mouseDown(e)
  end
  
  def rightMouseDown(e)
    self.superview.rightMouseDown(e)
  end
end

File: Classes/Views/TableProgressIndicator.m
// LimeChat is copyrighted free software by Satoshi Nakagawa <psychs AT limechat DOT net>.
// You can redistribute it and/or modify it under the terms of the GPL version 2 (see the file GPL.txt).

#import "TableProgressIndicator.h"


@implementation TableProgressIndicator

- (void)mouseDown:(NSEvent *)e
{
	[[self superview] mouseDown:e];
}

- (void)rightMouseDown:(NSEvent *)e
{
	[[self superview] rightMouseDown:e];
}

@end
